scripting/get5/surrender.sp

```diff
@@ -255,13 +223,9 @@ void CheckForForfeitOnDisconnect() {
   int team2Count = GetTeamPlayerCount(Get5Team_2);
 
   Get5Team forfeitingTeam;
-
-  if (g_GameState <= Get5State_Warmup && g_TeamTimeToStartCvar.IntValue > 0) {
-    // If we're in warmup or veto and a "time to ready" value is defined; let that handle it.
-    return;
-  } else if (team1Count > 0 && team2Count > 0) {
+  if (team1Count > 0 && team2Count > 0) {
     // If both teams still have at least one player; check for single-disconnect tech trigger.
-    if (g_GameState == Get5State_Live && g_AutoTechPauseCvar.IntValue == view_as<int>(Get5AutoTechPauseType_AnyPlayer)) {
+    if (g_AutoTechPauseCvar.IntValue == view_as<int>(Get5AutoTechPauseType_AnyPlayer)) {
```

Comment:
Reminder: Get rid of the `else` for this one and move `Get5Team forfeitingTeam;` below it. Readability is low here.